LEAF_Request_Portal/Email.php

```diff
@@ -527,10 +528,10 @@ function setSubjectWithTemplate($subjectTemplate)
     /**
      * Purpose (deprecated): set email body directly from passed in HTML
      * LEGACY: Included as scripts created by portal uses that implement sends using this feature
-     * @param $i
+     * @param string $i
      * @throws SmartyException
      */
-    public function setBody($i)
+    public function setBody(string $i): void
     {
```

Comment:
I don't see where the SmartyException is used unless it's within the $smarty->fetch, in which case are we supposed to show that in the PHPDoc of this method?